frontend/src/components/Footer/presenter.jsx

```
@@ -0,0 +1,16 @@
+import React from "react";
+import './style.css'
+import AdminPane from "../AdminPane";
+import {hasPermission} from "../common";
+import get from "lodash/get";
+
+export default function footer(props) {
```

ESLint

[error] Missing space before function parentheses.

(space-before-function-paren)